CHANGELOG.md

```diff
@@ -115,56 +129,84 @@ Early Access Packages
 
 - Renamed cash-settled European options (European -> EuropeanCash).
 
+- Requires
+
 #### Daml.Finance.Instrument.Swap
 
 - Dependencies update
 
 - Style changes
 
+- Requires
+
 #### Daml.Finance.Instrument.Token
 
 - Dependencies update
 
+- Requires
+
 #### Daml.Finance.Interface.Account
 
 - Dependencies update
 
+- Removed `type K = AccountKey`
+
+- Requires
+
 #### Daml.Finance.Interface.Claims
 
 - Dependencies update
 
+- Requires
+
 #### Daml.Finance.Interface.Data
 
 - Dependencies update
 
 - Style changes
 
+- Requires
+
 #### Daml.Finance.Interface.Holding
 
+- Dependencies update
+
 - Style changes
 
+- Requires
+
 #### Daml.Finance.Interface.Instrument.Base
 
 - Dependencies update
 
 - Style changes
 
+- Removed `type K = InstrumentKey`
+
+- Requires
+
 #### Daml.Finance.Interface.Instrument.Bond
 
 - Dependencies update
 
 - Added `GetView` to all instruments
 
+- Requires
+
 #### Daml.Finance.Interface.Instrument.Equity
 
 - Dependencies update
 
+- Requires
+
 #### Daml.Finance.Interface.Instrument.Generic
 
 - Dependencies update
 
 - Move Election logic from Generic to Lifecycle (to facilitate code reuse)
 
+- Requires
```

Comment:
I think we could be more descriptive, such as "removed `asDisclosure` method from instrument interface, added interface hierarchy" or something like that, wdyt?

Reply:
agree, will be more descriptive